README.md

```diff
@@ -8,3 +8,8 @@ If your space is missing in the [list](./directory.json), [fork the repository](
 add your space and create a pull request in GitHub.
 
 Find more information about the SpaceAPI on [spaceapi.io](https://spaceapi.io).
+
+Some tips for correct implemetation :
+* Check if your json file is valid with online [validator](https://spaceapi.io/validator/).
+* Tripple check your longatude/latatude if they are not swapped.
+* Check if CORS is enabled (adding a Access-Control-Allow-Origin: * header to the webserver).
```

Comment:
Maybe we could just point at https://spaceapi.io/provide-an-endpoint/ instead?

Reply:
Sure, we can do that if you prefer that. In my experience people read documentation poorly (and I saw in the pull request many times the same mistakes) so repeating a bit of documentation can't harm IMHO.